packages/driver/src/cy/chai.js

```diff
@@ -33,6 +34,9 @@ let existProto = null
 let getMessage = null
 let chaiUtils = null
 
+const LOGGED_ARRAY_SIZE = 10
+const LOGGED_OBJ_SIZE = 10
```

Comment:
I think this size is a bit small. We still want the UI to display a useful amount of information, especially when running in headless mode for screenshots. We just want to prevent it from crashing. 

I think the other issue with having this static length is that, couldn't they have infinite nested array or object so this ends up being quite a lot of data anyways? It's a good start though I guess.

Reply:
Sorry for the late reply. I was thinking about the solutions. 

I think limiting characters is the best solution. For example, count the characters of an item in an array/object and if the total string length goes over the limit (maybe 2000 characters), show `...more`. 

2000 characters are long enough and the maximum characters shown are around 4000 (1999 until the last item and about 2000 in the last).

I also thought about creating a array/object explorer for big objects. When an array/object is so big that `...more` is shown, click the text and create a popup and you can check the values by searching them.